cpu/lpc2387/periph/spi.c

```diff
@@ -82,6 +78,11 @@ int spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
     /*  power on */
     PCONP |= (PCSSP0);
 
+    spi_init_pins(bus);
```

Comment:
This seems weird to have gpio initialization in the spi_aquire function. Could it be kept in spi_init ?

Reply:
It seems to still work - I tested sending 100 pings and got 100 matching pongs :-)

This PR is updated with the pin initialization moved back to spi_init